php-practice/section5.php

```diff
@@ -320,3 +338,25 @@ public function send() {
  *   'cost' => 3.99
  * ];
  */
+
+class ShoppingCart {
+  private $item;
+  private $shoppingCart;
+
+  $item = [
+    'name' => 'Pringles',
+    'cost' => 3.99
+  ];
+
+  public function addItem() {
+    $shoppingCart =  [];
+    $shoppingCart[] = array_push($item);
```

Comment:
You're trying to get or set the `shoppingCart` property, so you need to use the `$this` reference to the current object:

```
$this->shoppingCart = $someValue;
$this->shoppingCart[] = $someOtherValue;
```